toolkits/global/packages/global-author-list/README.md

```diff
@@ -54,18 +59,76 @@ Overrides the default ` & ` with a `...` before the final item in the list.
+
+It is down to your application to decide whether or not to have the heading
+visibily hidden (it needs to be accessible to screen readers).
 
-It is down to your application to render the correct authors for the `--truncated` option to make sense.
+#### Javascript
 
-For example:
+The `authorLists()` function comes in handy to initialise all lists present in the
+document. 
 
-```kotlin
-val isTruncated = fullContributorList.size > 3
-val contributors = if (isTruncated) { fullContributorList.take(2) +  fullContributorList.last() } else fullContributorList
+```javascript
+import {authorLists} from '@springernature/global-author-list/js';
+
+authorLists(/* options */);
 ```
 
-```handlebars
-<ul class="c-author-list{{#if isTruncated}} c-author-list--truncated{{/if}}>...</ul> 
+For more flexibility you can use the `authorList()` function directly:
+
+```javascript
+import {authorList} from '@springernature/global-author-list/js/author-list';
+
+const myAuthorListContainer = document.querySelector('.my-author-list-container');
+
+authorList(myAuthorListContainer/*, options*/).init();
 ```
 
+#### Options
+
+| Option            | Type   | Default value                | Description                                                              |
+|-------------------|--------|------------------------------|--------------------------------------------------------------------------|
+| headingSelector   | String | '[data-author-list-heading]' | Selector to the author list's heading element                            |
+| listSelector      | String | 'ul'                         | Selector to the author list's list element                               |
+| authorHideClass   | String | 'c-author-list__hide'        | CSS class to hide items from the author list's list when it is truncated |
+| truncatedClass    | String | 'c-author-list--truncated'   | CSS class to toggle onto the list element when it is truncated           |
+| listModifierClass | String | null                         | CSS class to add to the list the component is initialised                |
+| buttonClassList   | String | null                         | List of CSS classes to style the toggle button                           |
+
+The data attribute options are the same, but are lowercase and hyphenated:
+
+- `data-author-list-heading-selector`
+- `data-author-list-list-selector`
+- `data-author-list-author-hide-class`
+- `data-author-list-truncated-class`
+- `data-author-list-list-modifier-class`
+- `data-author-list-button-class-list`
+
+Note: data attribute options will take precedence over any options set during initialisation.
+
+##### Heads up!
+
+- It is down to your application to toggle the `.js` class onto the `<html>`
+element, if you use the default `authorHideClass` value.
+
+- The JS component make use of an SVG image as icon in the toggle button that
+  swaps from `+` to `-`. It is down to your application to include these `+` and
+  `-` icons in its SVG sprite.
```

Comment:
```suggestion
> **NOTE**: The JS component make use of an SVG image as icon in the toggle button that
  swaps from `+` to `-`. It is down to your application to include these `+` and
  `-` icons in its SVG sprite.
```

Reply:
We are adding a link to the icons folder where the svg files are